pyproject.toml

```
@@ -51,6 +51,10 @@ doc = [
     "sphinx-autobuild",
     "sphinxcontrib-bibtex",
     "sphinxemoji",
+    # not needed directly, but ends up
+    # in dependency graph and lower
+    # versions cause issues
+    "docutils>=0.20.0",
```

Do you know what's the extension that doesn't support `docutils>0.18`?

---

It looks like it's sphinxcontrib-bibtex:

https://github.com/mcmtroffaes/sphinxcontrib-bibtex/commit/ba613822a65cd685e02cc2692afc1dd0efa8b90f (with follow up commit to fix error: https://github.com/mcmtroffaes/sphinxcontrib-bibtex/commit/f60c7cec6895674422ff8cdc85cdc757a31fb925 )

Although I must confess I haven't looked fully into what exactly causes the weird html output as I was sufficiently satisfied this was the problem.

Shall I edit the comment to something more specific, like:

```suggestion
    # not needed directly, but 
    # lower versions cause issues with
    # sphinxcontrib-bibtex, see
    # https://github.com/mcmtroffaes/sphinxcontrib-bibtex/commit/ba613822a65cd685e02cc2692afc1dd0efa8b90f
    "docutils>=0.20.0",
```

of course, an alternative is to just add a minimum version of `sphinxcontrib-bibtex` of `2.6.0` for the docs, where they have the relevant version pin for docutils themselves. However, that does exclude versions of `sphinxcontrib-bibtex` that work correctly (as long as they get the right version of docutils). What do you think?

---

Pinning to 2.6.0 looks like a clearer approach to me 👍🏼 . You can explain this in a comment next to this pin.

---

Done! sorry for the delay, I'm on holiday this week, but should be able to keep on top of this now.

---

Enjoy your holidays! This can definitely wait. There is no rush 😉